product.json

```diff
@@ -78,12 +78,6 @@
 	"extensionsGallery": {
 		"serviceUrl": "https://sqlopsextensions.blob.core.windows.net/marketplace/v1/extensionsGallery.json"
 	},
-	"builtInExtensions": [
-		{
-			"name": "Microsoft.sqlservernotebook",
```

Comment:
@chlafreniere do you have context on if it's fine to remove this extension reference?